cpp/kiss_icp/core/VoxelHashMap.cpp

```diff
@@ -75,7 +75,7 @@ void VoxelHashMap::Update(const std::vector<Eigen::Vector3d> &points, const Soph
 
 void VoxelHashMap::AddPoints(const std::vector<Eigen::Vector3d> &points) {
     std::for_each(points.cbegin(), points.cend(), [&](const auto &point) {
-        auto voxel = PointToVoxel(point);
+        auto voxel = PointToVoxel(point, voxel_size_);
```

Comment:
This is great, as its more explicit. `PointToVoxel(point)` sounds like you magically convert a point to a voxel; without the explicit `voxel_size,`, this doesn't make sense to read. :clap: